package.json

```diff
@@ -3,7 +3,7 @@
   "name": "min-vid",
   "version": "0.1.9",
   "private": true,
-  "icon": "resource://min-vid/docs/images/gradient-logo.png",
+  "icon": "resource://min-vid/docs/images/dark-logo.png",
```

Comment:
It isn't in the bundled XPI:

<img width="719" alt="index_of_jar_file____var_folders_qc_g_f0szh95k1fzph34rhmf5jw0000gp_t_f8476a30-a1ee-4ee7-b6a7-7b90e7dff2e3_extensions__min-vid_xpi__docs_images_" src="https://cloud.githubusercontent.com/assets/557895/18753526/77f60d7c-809a-11e6-87b9-b6095e9237b0.png">